notebook/agentchat_compression.ipynb

```diff
@@ -0,0 +1,927 @@
+{
```

Comment:
Just one note: There's no explanation on how the compression agent decides what to retain and what to omit, knowing the logic and reasoning would be beneficial to have here. Perhaps give a brief overview of what the default prompt does in the notebook?

Reply:
There is already description for the logic behind the compression, and I have revised them.